### PR TITLE
Econometric modelling script v1

### PR DESCRIPTION
**Pull request by pwisniewski-econ**
Publish script for econometric modelling

### CODE REVIEW

File: src/econometrics.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+control_ze = aggregate_data(control, ['ZE', 'LIB_ZE'], agg_dict_control)
+control_epci = aggregate_data(control, ['EPCI', 'LIB_EPCI'], agg_dict_control)
+
+# Processing for immo_reg_com
+immo_reg_com = merge_and_transform(immo_com, irflation, control, 'COM')
+immo_reg_com_diff = group_and_calculate_diff(immo_reg_com, ['COM'])
+
+# Total 
+immo_reg_corr =immo_reg_com_diff[["prixM2", "ir","lag_ir","lag2_ir","lag4_ir","popdensity2019", "med_change", "Physicist_access", "Q219", "prop_maison", "assault_for_1000", "dens_change"]].dropna().corr()
+corplot(immo_reg_corr)
```
Comment (lu-julia) [inline src/econometrics.py]: There is an error message here : corplot() is missing one argument : 'nom'

**Review by lu-julia — CHANGES_REQUESTED** [review]
Thanks for your work! Just a minor change needed before we can merge

File: src/econometrics.py (hunk 1 above)
Comment (pwisniewski-econ) [reply src/econometrics.py]: indeed! good catch

**Review by RemiCSK — APPROVED** [review]
Merge

**Review by lu-julia — APPROVED** [review]
Perfect

**Pull request merged**